shared/src/proxies/public/getPublicPractitionersByNameProxy.ts

```diff
@@ -0,0 +1,22 @@
+import { PractitionersByName } from '@web-api/business/useCases/practitioner/getPractitionersByNameInteractor';
+import { get } from '../requests';
+
+export const getPublicPractitionersByNameInteractor = (
+  applicationContext: IApplicationContext,
```

Comment:
Should this be `ClientApplicationContext`?

Reply:
And, come to think of it, should this go in `web-client` instead of `shared` too?

Reply:
All our Proxy files live in shared folder, and we use IApplicationContext when using "GET" HTTP request